src/handbook/peopleops/organization.md

```diff
@@ -123,3 +123,15 @@ a PIP. This ensures that a PIP is a genuine change.
 If an employee's performance does not improve, or if they violate the terms of
 their PIP, they may be terminated. Termination will be based on a documented
 pattern of underperformance or unacceptable behavior.
+
+## CEO underperformance
+
+As with all employees in the company the CEO can underperform. Generally speaking
+the CEO does get performance reviews, though these are provided by the Board of
+Directors.
+
+All employees are free to provide feedback or input to the board of directors
```

Comment:
Going to the board as an employee about CEO performance feels very last port of call and serious though?

Even for what might be general performance concerns where there could be areas of improvement for example?

Reply:
For example, a single employees grievances about CEO performance may just be that single employee, but going to the board feels very serious, particularly in a case like this?

As someone that's never operated on a board, I don't really know _how_ these kind of things are received by a board.

Reply:
It should be as serious as every other time discussing performance to someone's manager and not them.

I'll update this PR